manim/utils/space_ops.py

```diff
@@ -231,7 +233,7 @@ def angle_of_vector(vector):
         return np.angle(complex(*vector[:2]))
 
 
-def angle_between_vectors(v1, v2):
+def angle_between_vectors(v1: tuple, v2: tuple) -> np.float64:
```

Comment:
why float64 and not only float?

Reply:
And why tuple and not Sequence[float]?
See first point here:
https://docs.manim.community/en/latest/contributing/documentation.html#adding-type-hints-to-functions-and-parameters

Reply:
The reason I wrote like this was because I got these as the type when I checked the function variable with `type()` and printed them out.

Reply:
I see, can you maybe post the example here how you did that?
Then I will try to find a counter-example, to make this type-hint bulletproof :)

Reply:
I did one more edit for the whole file, can you check and let me if there is anything wrong with it?